### PR TITLE
Use SSL by default; add option to turn it off

### PR DESCRIPTION
**Pull request by timmywil**
I'd like to use SSL as before, and it's more difficult to enable SSL than it is to disable it. I've reverted the change to use port `9412` without SSL by default, but I made it so it can easily be added back with **one** change:

### `.env`
```
JQUERY_WP_HTTP_PORT=9412
```

This port is respected in the new `wp-config.php`, so no further changes required. I also made it so the virtual host for 443 is only created in apache if the certs are available in the SSL folder. I needed to keep `443:443` in the docker compose (conditionals are not possible and I don't want to have to construct the docker compose), but without certs, there's nothing listening on 443. I don't *think* that will be an issue for anyone, but let me know. Even if it is, however, it can be changed in the .env as `JQUERY_WP_HTTPS_PORT` to `0` to get some random, unused port.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,13 +105,21 @@ docker exec -it jquerydb mysql -u root -proot
 
 ### Ports
 
-If you already use port 9412 on your host, you can create a `.env` file in this directory and set the following environment variable with a port number of your own choosing:
```
Comment (mgol) [inline README.md:70]: Port 9412 was added by @Krinkle on purpose, to not conflict with locally running nginx which I had issues with.

File: docker-compose.yml [hunk 2]
```diff
@@ -31,7 +34,7 @@ services:
     volumes:
       - jquerydbdata:/var/lib/mysql
     ports:
-      - "${JQUERY_WP_MYSQL_PORT:-9414}:3306"
+      - "3306:3306"
```
Comment (Krinkle) [inline docker-compose.yml]: 3306 is the default MySQL port. If you install mysql (e.g. apt-get or brew), or have any other project with a similar docker container, this immediately clashes.

Unlike http/https, the mysql port doesn't actually need to be exposed to the outside. We could remove it completely, or (my suggestion) to leave it with a custom value as the default.

It's sole purpose is to allow you to explore the database through some GUI app or CLI tool that you fancy from your own machine to be able to connect to it.

File: README.md (hunk 1 above)
Comment (Krinkle) [reply README.md:70]: Indeed, I would much prefer not to bind port 80 or 8080 by default, but something more personal like 9412.

The problem with a custom port, however, is that you can't have both HTTP and HTTPS on the same port. And it'd require a ton of refactoring and logic to try to maintain a working web server and theme logic that is able to serve different-but-functionally-the-same HTML over entirely different ports. You can't simply generate protocol-relative URLs to solve it, instead the code would have to be aware of both sets of ports. This is what led me to get rid of it.

**Comment by Krinkle** [comment]
I've left comments indicating the issues, but those are solvable.

Out of curiosity, what is the reason for preferring HTTPS locally? Does it allow better testing of certain pages?

File: README.md (hunk 1 above)
Comment (timmywil) [reply README.md:70]: Yes, 9412 was added so it wouldn't conflict with your ports, which may include 8080, but I didn't want SSL support removed. This PR adds it back, but in a way that makes it easy to disable. It shouldn't affect you once you add the `.env` file setting `JQUERY_WP_MYSQL_PORT`.

> You can't simply generate protocol-relative URLs to solve it

That is why I enabled it by default. As I said in the issue, it's a lot easier to disable SSL than it is to add it.

File: docker-compose.yml (hunk 2 above)
Comment (timmywil) [reply docker-compose.yml]: Actually, not exposing it at all makes the most sense to me.

**Comment by timmywil** [comment]
> Out of curiosity, what is the reason for preferring HTTPS locally?

We talked about it a bit in chat. Essentially, my security settings (mainly my VPN), make it difficult to allow non-https anywhere but localhost. The settings are admittedly aggressive, but it's easier to support it here than it is for me to change it. Also, there are advantages to being able to test these sites locally with https. In the immediate future, it will come in handy when testing CSP changes.

**Review by mgol — APPROVED** [review]
I confirm setting `JQUERY_WP_HTTP_PORT` works so I have no further reservations against this PR.

**Pull request merged**